examples/SerialIface/SerialIface.ino

```diff
@@ -85,21 +95,36 @@ void processBinaryCmds() {
   }
 }
 
+void processFastBinaryCmds() {
+  while (Serial.available() >= FAST_BINARY_CMD_SIZE) {
+    byte cmd[FAST_BINARY_CMD_SIZE];
```

Comment:
This could be global so you're not re-creating the array on every loop

Reply:
I don't think there's a cost or disadvantage to this being a local variable. I haven't looked at the disassembled code, but making it global probably just means that the buffer would be in a fixed memory location, instead of on the stack. In my opinion, being a local variable makes it clear that it's only used in this function.